include/multiverso/table/matrix_table.h

```diff
@@ -29,16 +29,16 @@ class MatrixWorkerTable : public WorkerTable {
   void Get(const std::vector<integer_t>& row_ids,
            const std::vector<T*>& data_vec, size_t size);
 
-  // Add whole table
-  void Add(T* data, size_t size, const AddOption* option = nullptr);
-
   void Add(integer_t row_id, T* data, size_t size, 
            const AddOption* option = nullptr);
 
   void Add(const std::vector<integer_t>& row_ids,
            const std::vector<T*>& data_vec, size_t size, 
            const AddOption* option = nullptr);
 
+  void Add(T* data, size_t size, const AddOption* option = nullptr,
+           integer_t* row_ids = nullptr, integer_t row_ids_size = 0);
```

Comment:
what if the memory for these row ids are not continuous? The new interface seems assume it's a continuous memory for each rows.